### PR TITLE
version bump and changelog

### PR DESCRIPTION
**Pull request by KevinMenden**
Bump version and write changes to changelog for 2.0.0 release

sorry for abusing the template merge branch for this :grin:  will delete that one afterwards anyway

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,14 +3,23 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v2.0.0dev - [date]
+## 2.0.0 Silver Zebra - [2021-05-12]
```
Comment (mashehu) [inline CHANGELOG.md]: As a naming scheme I was thinking of `metal`+`animal classically kept in cages`. I am not sure zebra fall into that category 🤔

File: nextflow.config [hunk 2]
```diff
@@ -201,7 +201,7 @@ manifest {
   description = 'CAGE-seq pipeline'
   mainScript = 'main.nf'
   nextflowVersion = '>=21.04.0'
-  version = '2.0.0dev'
+  version = '2.0.0'
```
Comment (mashehu) [inline nextflow.config:204]: are you sure this shouldn't still be `dev`?

**Review by mashehu — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (KevinMenden) [reply CHANGELOG.md]: Aaaah good idea :rofl: 
You choose one then :)

File: nextflow.config (hunk 2 above)
Comment (KevinMenden) [reply nextflow.config:204]: Not if we release it, right?
Do we want to include anything else, or should we try to release the 2.0.0 version?

File: CHANGELOG.md (hunk 1 above)
Comment (mashehu) [reply CHANGELOG.md]: The codename generator oracle has spoken 😀:
```suggestion
## 2.0.0 Nickel Chicken - [2021-05-12]
```

File: nextflow.config (hunk 2 above)
Comment (mashehu) [reply nextflow.config:204]: Ah, yes, sorry. I always forget the order of when to switch 🙃

File: CHANGELOG.md (hunk 1 above)
Comment (KevinMenden) [reply CHANGELOG.md]: Nice :grin:

**Pull request merged**